lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -42,8 +42,9 @@ __PACKAGE__->add_columns(
         data_type      => 'integer',
         is_foreign_key => 1,
     },
-    hidden => {
-        data_type     => 'boolean',
+    flags => {
```

Comment:
Not related at all, I wanted to do one db schema change instead of multiple. I didn't find any place where we use this "hidden" column, that's why I didn't bother with migration. Did I miss someplace?